test/shared/identityUtils.test.ts

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import fs from 'node:fs';
+import { expect } from 'chai';
+import { TestContext } from '@salesforce/core/testSetup';
+import { DevServerUtils } from '../../src/shared/devServerUtils.js';
+import { IdentityUtils } from '../../src/shared/identityUtils.js';
+
+describe('identityUtils', () => {
+  const $$ = new TestContext();
+
+  afterEach(() => {
+    $$.restore();
+  });
+
+  it('createIdentityToken resolves if lwr.config.json is not found', async () => {
```

Comment:
Depending on how we decide to update `createIdentityToken` the tests here would need to be updated.